timers.js

```diff
@@ -1,4 +1,43 @@
+var timer = {
+  seconds:0,
+  timerId:0,
+}
+//step one create JavaScript Selectors
+document.getElementById("start");
+document.getElementById("pause");
+document.getElementById("reset");
 
+//Create click handlers rough code started hit time cutoff
+document.getElementById("start").addEventListener("click", function(){
+    document.getElementById("begin"). = "";
+});
 
+  document.getElementById("pause").addEventListener("click", function(){
+    document.getElementById(""). = "";
+  });
 
+    document.getElementById("reset").addEventListener("click", function(){
+    document.getElementById(""). = "";
+    });
 
+//pasted in solution to try and see how it worked - and try to make it work in a diff way
```

Comment:
Totally fine that you played around with the solution, but I would refrain from leaving it in the submitted assignment.